src/commands/auth.json

```diff
@@ -0,0 +1,41 @@
+{
+    "AUTH": {
+        "summary": "Authenticate to the server",
+        "complexity": "O(N) where N is the number of passwords defined for the user",
+        "group": "connection",
+        "since": "1.0.0",
+        "arity": -2,
+        "function": "authCommand",
+        "history": [
+            [
+                "6.0.0",
+                "Added ACL style (username and password)."
```

Comment:
so "history" is an array of pairs?
i.e. allows multiple changes per version..

Reply:
yes, is that ok with you?

Reply:
i suppose.. it can't be a map.
the alternative is that each version holds an array of changes.
i don't mind that much.